### ISSUE TITLE
How to create a widget to show followed items?

### ISSUE THREAD

**Issue by giannik** [siyamandayubi/Orchardcollaboration#1]
Currently I can follow discussions and wiki pages. Is there a widget  to show items I am following ?
Thanks.

**Comment by siyamandayubi** [comment]
No not at the moment. The followers are stored in the ContentItem and they are not persisted in their own table which in terms of performance in loading the ContentItem is good, but it makes it difficult to run a query that give us the list of items that I am following. We can have a special widget that build an HqlQuery that searches for userId in the FollowerPart. It would not be nice & best practice, but it would work.

**Comment by giannik** [comment]
Ok, I get it.
Thanks.

**Comment by giannik** [comment]
So , I guess the benefit of following an item is to receive notifications if something changes ? Correct?

**Comment by siyamandayubi** [comment]
Yep, that is true. Also it would be trivial to represent list of people who follow an item.
